source/guide_fluentd.rst

```diff
@@ -0,0 +1,192 @@
+consumption for a better use and understanding of data.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`Ruby <lang-ruby>` and its package manager :manual_anchor:`gem <lang-ruby.html#gem>`
+  * :manual:`supervisord <daemons-supervisord>`
+  * Optional: :manual:`web backends <web-backends>`
+
+License
+=======
+
+Fluentd_ is released under the Apache-2.0 License.
+
+All relevant legal information can be found here
+
+  * https://github.com/fluent/fluentd/blob/master/LICENSE
+  * http://www.apache.org/licenses/LICENSE-2.0
+
+Prerequisites
+=============
+
+We're using :manual:`Ruby <lang-ruby>` in the stable version 2.6:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version show ruby
+ Using 'Ruby' version: '2.6'
+ [isabell@stardust ~]$
+
+Installation
+============
+
+Fluentd_ can be installed as a ruby gem:
+
+::
+
+ [isabell@stardust ~]$ gem install fluentd --no-doc
+ Fetching http_parser.rb-0.6.0.gem
+ Fetching concurrent-ruby-1.1.7.gem
+ Fetching tzinfo-2.0.2.gem
+ Fetching serverengine-2.2.1.gem
+ Fetching sigdump-0.2.4.gem
+ Fetching yajl-ruby-1.4.1.gem
+ Fetching msgpack-1.3.3.gem
+ Fetching cool.io-1.6.0.gem
+ Fetching fluentd-1.11.2.gem
+ Fetching tzinfo-data-1.2020.1.gem
+ Fetching strptime-0.2.4.gem
+ WARNING:  You don't have /home/isabell/.gem/ruby/2.6.0/bin in your PATH, gem executables will not run.
+ Building native extensions. This could take a while...
+ Successfully installed msgpack-1.3.3
+ Building native extensions. This could take a while...
+ Successfully installed yajl-ruby-1.4.1
+ Building native extensions. This could take a while...
+ Successfully installed cool.io-1.6.0
+ Successfully installed sigdump-0.2.4
+ Successfully installed serverengine-2.2.1
+ Building native extensions. This could take a while...
+ Successfully installed http_parser.rb-0.6.0
+ Successfully installed concurrent-ruby-1.1.7
+ Successfully installed tzinfo-2.0.2
+ Successfully installed tzinfo-data-1.2020.1
+ Building native extensions. This could take a while...
+ Successfully installed strptime-0.2.4
+ Successfully installed fluentd-1.11.2
+ 11 gems installed
+ [isabell@stardust ~]$
+
+.. note:: You can ignore the warning about the executables not being in the PATH, Uberspace already took care of that.
+
+To verify the installation, check the version of fluentd:
+
+::
+
+ [isabell@stardust ~]$ fluentd --version
+ fluentd 1.11.2
+ [isabell@stardust ~]$
+
+Configuration
+=============
+
+Create configuration
+---------------------------
+Create the default configuration in the ``~/fluentd`` directory.
+
+::
+
+ [isabell@stardust ~]$ fluentd --setup ~/fluentd
+ Installed /home/isabell/fluentd/fluent.conf.
+ [isabell@stardust ~]$
+
+Edit this configuration and comment out the following lines:
+
+.. code-block::
+
+ # match tag=system.** and forward to another fluent server
+ <match system.**>
+   @type forward
```

Comment:
This receives data from another flutend and pushes it along, right? What do you think about a more illustrative example like sending `~/logs/supervisord.log` to another fluentd?

Reply:
Yes, that's correct. This is part of the default fluentd configuration, created by `fluentd --init`. I assumed, that the user does not have another fluentd instance set up and thus included the instruction to comment out this part of the configuration. Should I make this more clear by adding a `#` in front of each line?

Reply:
oh, whoops. I totally didn't get that while reading it, so probably, yes :sweat_smile: